src/xdp/trn_transit_xdp1_3layers.c

```diff
@@ -0,0 +1,733 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Can we rename this file to trn_transit_xdp_hardware_offload.c? 

And also please fix the description below detailing what functions are offloaded.

Has this been tested in non-offload mode (driver or generic)?

Reply:
Since [multiple programs are supported on the same XDP interface](https://xdp-project.net/#Multiple-XDP-programs-on-a-single-interface), our program works together with original Transit XDP Program.
Since Netronome NIC has bugs on XDP driver mode, we only try the combination about original Transit XDP Program (generic mode) and offload Transit XDP Program (offload mode).